packages/ckb-indexer/src/indexer.ts

```diff
@@ -51,6 +51,9 @@ export class CkbIndexer implements CellProvider, TerminableCellFetcher {
   medianTimeEmitters: EventEmitter[] = [];
   emitters: IndexerEmitter[] = [];
   isSubscribeRunning = false;
+
+  constructor(ckbRpcUrl: string);
```

Comment:
it says in https://github.com/ckb-js/lumos/pull/445
``` ts
import { Indexer } from '@ckb-lumos/lumos';

// before
// const indexer = new Indexer(
//   "https://testnet.ckb.dev/indexer", 
//   "https://testnet.ckb.dev/rpc",
// );

// after
const indexer = new Indexer("https://testnet.ckb.dev/rpc");
```

So when it have one parameter, the first parameter means `ckbRpcUrl`, not `ckbIndexerUrl`